scripts/periodic/410.pkg-audit.in

```diff
@@ -156,6 +157,19 @@ audit_pkgs_all() {
 	esac
 
 	for j in $jails ; do
+		# ignore some jails
+		if [ -n "$security_status_pkgaudit_jails_ignore" ]; then
+			# we iterate to get exact matches because we want substring matches
+			# foo should not match foo.bar
+			for ignore in $security_status_pkgaudit_jails_ignore ; do
```

Comment:
I don't think if [ -n ... ] is necessary, for ignore in ${security_status_pkgaudit_jails_ignore+} directly would to the same. This will eliminate one level of indentation

Reply:
Nice.  Done.  Thank you.